flow/packet-headers/snmpv2.yaml

```diff
@@ -0,0 +1,201 @@
+          type: string
+          x-enum:
+            no_error:
+              x-field-uid: 1
+            too_big:
+              x-field-uid: 2
+            no-such_name:
+              x-field-uid: 3
+            bad_value:
+              x-field-uid: 4
+            read_only:
+              x-field-uid: 5
+            gen_err:
+              x-field-uid: 6
+            no_access:
+              x-field-uid: 7
+            wrong_type:
+              x-field-uid: 8
+            wrong_length:
+              x-field-uid: 9
+            wrong_encoding:
+              x-field-uid: 10
+            wrong_value:
+              x-field-uid: 11
+            no_creation:
+              x-field-uid: 12
+            inconsistent_value:
+              x-field-uid: 13
+            resource_unavailable:
+              x-field-uid: 14
+            commit_failed:
+              x-field-uid: 15
+            undo_failed:
+              x-field-uid: 16
+            authorization_error:
+              x-field-uid: 17
+            not_writable:
+              x-field-uid: 18
+            inconsistent_name:
+              x-field-uid: 19
+          x-field-uid: 2
+        error_index:
+          description: >-
+            When Error Status is non-zero, 
+            this field contains a pointer that specifies which object generated the error. 
+            Always zero in a request.
+          type: integer
+          format: int64
+          minimum: 0
+          maximum: 2147483647
+          x-field-uid: 3
+        variable_bindings: 
+          description: >-
+            A Sequence of variable_bindings.
+          type: array
+          items:
+            $ref: '#/components/schemas/Flow.Snmpv2.VariableBinding'
+          x-field-uid: 4
+    Flow.Snmpv2.VariableBinding:
+      description: A Sequence of two fields, an object_identifier and the value for/from that object_identifier.
+      type: object
+      properties:
+        object_identifier:
+          description: >-
+              object_identifier that points to a particular parameter in the SNMP agent.
+          type: array
+          items:
+            type: integer
+            format: int64
+          x-field-uid: 1
+        value: 
+          $ref: '#/components/schemas/Flow.Snmpv2.VariableBindingValue'
+          x-field-uid: 2
+    Flow.Snmpv2.VariableBindingValue: 
+      description: >-
+          value for/from that object_identifier.
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          x-enum:
+            ip_address:
+              x-field-uid: 1
+            counter:
+              x-field-uid: 2
+            timeticks:
+              x-field-uid: 3
+            arbitrary:
+              x-field-uid: 4
+            big_counter:
+              x-field-uid: 5
+            unsigned_integer:
+              x-field-uid: 6
+        ip_address:
+          description: >-
+            octet string
```

Comment:
I understand it can contain both ipv4 and ipv6 address ? Maybe we can have better strict data types by asking for v4/v6 type and then specific address ?

Reply:
Added